src/components/Gallery/Gallery.tsx

```diff
@@ -0,0 +1,119 @@
+import React from "react";
+import { Image } from "../Image/Image";
+import { Swiper, SwiperSlide } from "swiper/react";
+import { Pagination } from "swiper/modules";
+import "swiper/css/pagination";
+import "swiper/css";
+
+import eventImage1 from "../../assets/event_diningTable.png";
+import eventImage2 from "../../assets/event_walledTent.png";
+import eventImage3 from "../../assets/event_conference.png";
+import eventImage4 from "../../assets/event_beachPopupTents.png";
+import eventImage5 from "../../assets/event_colorLights.png";
+import eventImage6 from "../../assets/event_treeLights.png";
+import eventImage7 from "../../assets/event_marathonFinishLine.png";
+import eventImage8 from "../../assets/event_groupOutdoor.png";
+import eventImage9 from "../../assets/event_SantaPhotos.png";
+
+export const Gallery: React.FC = () => {
+  const images = [
+    eventImage1,
+    eventImage2,
+    eventImage3,
+    eventImage4,
+    eventImage5,
+    eventImage6,
+    eventImage7,
+    eventImage8,
+    eventImage9,
+  ];
+
+  return (
+    <div className="gallery-container max-w-[1180px] mx-auto ">
+      {/* Wide screen layout (3x3 grid for large screens) */}
+      <div className="  sm:hidden md:hidden grid grid-cols-3 gap-5">
+        {images.map((src, index) => (
+          <Image
+            key={index}
+            src={src}
+            alt={`Event Image ${index + 1}`}
+            context="gallery"
+          />
+        ))}
+      </div>
+
+      {/* Carousel for small and medium screens */}
+      <div className="hidden sm:flex md:flex ">
+        <Swiper
+          pagination={{
+            type: "custom",
+            renderCustom: (swiper, currentPage, totalPages) => {
+              const paginationBullets = Array(totalPages)
+                .fill(null)
+                .map((_, index) => {
+                  const className =
+                    currentPage === index + 1
+                      ? "swiper-pagination-bullet-active"
+                      : "swiper-pagination-bullet";
+
+                  return `
+                    <span class="${className}"
+                      style="
```

Comment:
We have a lot of styles defined in this file, could we instead use some Tailwind styles?

Reply:
Okay yes I did my best to use Tailwind more than before, however there were a couple areas where tailwind didn't seems to override the package styling and I had to use inline. I will push the changes soon to the new branch